src/components/Post/GovernanceSideBar/Referenda/VoteReferendumEthV2.tsx

```diff
@@ -82,11 +84,15 @@ const VoteReferendumEthV2 = ({ className, referendumId, onAccountChange, lastVot
 	const [abstainVoteValue, setAbstainVoteValue] = useState<BN>(ZERO_BN);
 	const [ayeVoteValue, setAyeVoteValue] = useState<BN>(ZERO_BN);
 	const [nayVoteValue, setNayVoteValue] = useState<BN>(ZERO_BN);
-
 	const [availableBalance, setAvailableBalance] = useState<BN>(ZERO_BN);
 	const [balanceErr, setBalanceErr] = useState('');
-
 	const [vote,setVote] = useState< EVoteDecisionType>(EVoteDecisionType.AYE);
+	const[ayeVoteVal,setAyeVoteVal] = useState<BN>(ZERO_BN);
+	const[nayVoteVal,setNayVoteVal] = useState<BN>(ZERO_BN);
+	const[abstainVoteVal,setAbstainVoteVal] = useState<BN>(ZERO_BN);
+	const[totalVoteVal,setTotalVoteVal] = useState<BN>(ZERO_BN);
```

Comment:
totalVoteVal remove that use direct value

Reply:
resolved